TASVideos/Pages/AwardsEditor/Assign.cshtml.cs

```diff
@@ -90,7 +106,7 @@ public async Task<IActionResult> OnPostRevoke(string shortName)
 	{
 		var awardToRevoke = (await _awards
 			.ForYear(Year))
-			.SingleOrDefault(a => a.ShortName == shortName);
+			.FirstOrDefault(a => a.ShortName == shortName);
```

Comment:
Now here's the part that fixes the crash (in those problematic cases we have the same award twice, because it was mistakenly assigned to a user and to a movie), but it got this comment on discord:

> also I don't think you should ever use First/FirstOrDefault if you don't have an ordering. I think EF Core even warns you about that (if executed and you look into the logs). First is "safer" than Single in the sense that it doesn't crash if there's duplicates, but I think you actually want to crash if there's something wrong